lua/lib/renderer.lua

```diff
@@ -71,13 +71,19 @@ if vim.g.lua_tree_git_hl == 1 then
     ["??"] = { { hl = "LuaTreeFileNew" } },
     ["R "] = { { hl = "LuaTreeFileRenamed" } },
     ["UU"] = { { hl = "LuaTreeFileMerge" } },
+    [" D"] = { { hl = "LuaTreeFileDeleted" } },
     dirty = { { hl = "LuaTreeFileDirty" } },
   }
   get_git_hl = function(node)
     local git_status = node.git_status
     if not git_status then return end
 
     local icons = git_hl[git_status]
+
+    if icons == nil then
+      icons = git_hl.dirty
+    end
+
```

Comment:
i think i prefer to get an error whenever we encounter a git state we don't have :)

Reply:
What do you think about a warning message alongside the fallback ? If it errors out it completely breaks the tree, it's a bit annoying when it happens.

Reply:
I think it might be a better solution indeed. Just print the git state that wasn't recognized (and maybe a suggestion to the user to open an issue with the warning message).